src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
I check for `extension.inIncognitoContext` in a content script and use it to set `pb.inIncognito`. Then changed `_syncStorage`, the way you mention [here](https://github.com/cowlicks/privacybadgerchrome/blob/firefox-compat/src/storage.js#L406-L410). This works around the webextension bug.

But things are still broken, changes I make to PB in FF private mode, still effect PB in regular FF.

To make this work without "split" mode, would we just have to guard other places we use storage with checks to `inIncognitoContext`? Or is it a larger issue?